feat(wiki): add an introduction to creating jobs (#45)
OpenJobDescription/openjd-specifications: wiki/Home.md

## Getting Started

The fastest way to understand the bones of Open Job Description is to understand both [How Jobs Are Constructed](How-Jobs-Are-Constructed)

> and [How Jobs Are Run](How-Jobs-Are-Run). Then, once you've given those a look through go through the
> [Introduction to Creating a Job](Introduction-to-Creating-a-Job) and then browse the provided

Reviewer: Little awkward. Try

```suggestion
and [How Jobs Are Run](How-Jobs-Are-Run), followed by the [Introduction to Creating a Job](Introduction-to-Creating-a-Job) and browsing the provided
```
